Properties.cpp

```diff
@@ -100,22 +100,25 @@ inline void readProperty(PropertiesStore const &store,
         return;
     }
     auto &val = store.getVariant(batchEntry.prop);
-    auto containedTag = val.apply_visitor(GetContainedTypeTag());
+    auto containedTypeTag = GetContainedTypeTag();
```

Comment:
This perhaps could be better named `visitor` but that's nit-picking - thanks for catching MSVC being lax with passing a temporary as an l-value!